docs/installationGuide.rst

```diff
@@ -58,7 +58,7 @@ Roddy 2.3
 
   - If you intend to use Roddy and do not want to develop plugins or Roddy itself:
 
-    - Download any `JRE v1.8.*`_ (OpenJDK and SunJDK were tested). Also download `Groovy 2.3.*`_
+    - Download any `JRE v1.8.*`_ (OpenJDK and SunJDK were tested). Also download `Groovy 2.4.*`_
```

Comment:
I don't recall, if we used 2.4 in Roddy 2.3...

Reply:
You are right. This changed toward roddy 2.4/3.